db/re/skill_tree.txt

```diff
@@ -2612,6 +2612,8 @@
 4057,2515,5,2044,1,2053,1,0,0,0,0,0,0 //AB_SECRAMENT#Secrament#
 4057,5011,5,2051,2,0,0,0,0,0,0,0,0 //AB_OFFERTORIUM#Offertorium#
 4057,5014,5,0,0,0,0,0,0,0,0,0,0 //ALL_FULL_THROTTLE#Full Throttle#
+4057,5072,5,2044,1,2053,1,0,0,0,0,0,0 //AB_VITUPERATUM#Vituperatum#
+4057,5073,1,2031,1,2046,5,0,0,0,0,0,0 //AB_CONVENIO#Convenio#
```

Comment:
2031 should be 2039

Reply:
You're right, but it should be 2046, not 2039.